dbt/models/default/default.vw_pin_status.sql

```diff
@@ -159,7 +162,7 @@ LEFT JOIN ({{ aggregate_cdu(
     AND pdat.taxyr = cdat.taxyr
 LEFT JOIN ({{ aggregate_cdu(
     source_model = source('iasworld', 'dweldat'),
-    cdu_column = 'cdu'
+    cdu_column = 'user16'
```

Comment:
user16 seems to be the alternative cdu code in each table

Reply:
**[Question, non-blocking]** I don't have the context for why we're making this change -- have we just been pulling CDUs from the wrong `dweldat` column this whole time?

Reply:
Ah right, I forgot this view was so new. Thanks!